src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @OneToOne(fetch = FetchType.LAZY)
+    private User user;
```

Comment:
The `@OneToOne` relationship with `User` is set to `FetchType.LAZY`. Ensure that lazy loading is appropriate for your use case, as accessing the user might require an additional database query if it's not already loaded.